x-pack/plugins/fleet/server/services/preconfiguration.ts

```diff
@@ -66,11 +74,26 @@ export async function ensurePreconfiguredPackagesAndPolicies(
   }
 
   // Preinstall packages specified in Kibana config
-  const preconfiguredPackages = await Promise.all(
-    packages.map(({ name, version }) =>
-      ensureInstalledPreconfiguredPackage(soClient, esClient, name, version)
-    )
-  );
+  const preconfiguredPackages = await bulkInstallPackages({
+    savedObjectsClient: soClient,
+    esClient,
+    packagesToInstall: packages.map((pkg) =>
+      pkg.version === PRECONFIGURATION_LATEST_KEYWORD ? pkg.name : pkg
+    ),
+    force: true, // Always force outdated packages to be installed if a later version isn't installed
```

Comment:
I don't think the force flag in `installPackageFromRegistry` works like this? this _will_ allow an older version to be installed even when a newer version is already installed, which we don't want for preconfigured packages (right?). at the same time we do want to allow an older version to be installed even when a newer version _exists_ (but is not installed).

Reply:
:+1: Forgot to port over the version guard from `ensureInstalledPackage`. Just pushed a fix.